app/loader.js

```diff
@@ -7,62 +7,61 @@ Loads all the scripts and binaries data from a module package.json
 const path = require('path');
 const fs = require('fs');
 
+class Loader{
+    static getContents(content, dir) {
+        return {
+            dir,
```

Comment:
This is correct, but personally I prefer to explicitly state the name (`dir:dir`)